DrinkManagerConsole/SearchDrinkConsoleUi.cs

```diff
@@ -128,5 +128,68 @@ public static void AddMoreDrinksFromFile(List<Drink> drinksList)
             Console.WriteLine("\nPress any key to go back to the main menu.");
             Console.ReadKey();
         }
+        /// <summary>
+        /// Shows search criteria menu, gets user input and cause GetDrinksByAlcoholContent to run
+        /// </summary>
+        /// <param name="drinks"></param>
+        public static void HandleSearchDrinksByContentInConsole(List<Drink> drinks)
+        {
+            Console.Clear();
+            Console.WriteLine("Choose one of the searching criteria: ");
+            Console.WriteLine("1. Alcoholic drinks");
+            Console.WriteLine("2. Non alcoholic drinks");
+            Console.WriteLine("3. Optional alcohol drinks");
+            Console.WriteLine("4. Alcoholic and optional alcohol drinks");
+            Console.WriteLine("5. Non alcoholic and optional alcohol drinks");
+            Console.WriteLine("Press any other key to go back to previous menu");
+
+            var searchChoice = Console.ReadKey();
+            if (searchChoice.Key == ConsoleKey.D1 || searchChoice.Key == ConsoleKey.D2 || searchChoice.Key == ConsoleKey.D3 || searchChoice.Key == ConsoleKey.D4 || searchChoice.Key == ConsoleKey.D5)
+            {
+                DisplaySearchResults(GetDrinksByAlcoholContent(searchChoice, drinks));
+            }
+        }
+        /// <summary>
+        /// Depending on user input from HandleSearchDrinksByContentinConsole, causes SearchByAlcoholContent to run with specified search criteria
+        /// </summary>
+        /// <param name="key"></param>
+        /// <param name="drinks"></param>
+        /// <returns></returns>
+        public static List<Drink> GetDrinksByAlcoholContent(ConsoleKeyInfo key, List<Drink> drinks)
+        {
+            var contemporaryList = new List<Drink>();
+            Console.Clear();
+            switch (key.Key)
```

Comment:
I think that this switch statement could only determine the string that would later be passed to the SearchByAlcoholContent method, this way we would call that method after the switch.

Reply:
I agree, but as I mentioned in previous comment - because of 4th and 5th search options I think this way is unevitable (at least for now). I'll make some improvements in this field tomorrow

Reply:
We can as well reduce this search to 3 options and implement improvements you were talking about, and later I will try to add 4th and 5th option using another way.